plugins/ibm_qradar/plugin.spec.yaml

```
@@ -974,12 +999,19 @@ actions:
         required: false
         description: Specify the list of fields to be returned in the response. Specify the subfields in parentheses. Multiple fields in the same object must be comma separated. Sample fields to filter are id, create_time, username, note_text. More information about the fields can be found in plugin documentation
         example: id, note_text
+      filter:
+        title: Filter
+        type: string
+        required: false
+        description: Restrict the elements in a list based on the contents of various fields. E.g. id=55 and is_deleted = false
```

You could do away with this if we're already providing an example
```suggestion
        description: Restrict the elements in a list based on the contents of various fields
```